src/storage/mod.rs

```diff
@@ -34,6 +35,7 @@ use self::kv::with_tls_engine;
 use self::metrics::*;
 use self::mvcc::{Lock, TsSet};
 use self::txn::scheduler::Scheduler as TxnScheduler;
+use crate::server::lock_manager::{DummyLockMgr, LockMgr};
```

Comment:
Oh, `server` shouldn't be `storage`'s dependency, I think. `server` combines components together, and `storage` may be separated to a workspace in the future.

Reply:
OK, I'll shuffle stuff around a bit...